### DISCUSSION TITLE
Batch Audio to Text Convert.  Drag-and-drop + working with file list

### DISCUSSION THREAD

**Discussion by ard-av** [SubtitleEdit/subtitleedit#6296] [Ideas]
![Screenshot_1](https://user-images.githubusercontent.com/114281218/193201527-e23dc24e-af97-42c7-b4ef-258dc862c89d.jpg)

Please add
**Batch Audio to Text Convert** **Drag-and-drop** + **working with file list**.
Thanks

**Comment by niksedk** [comment]
OK, hopefully working now? https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.7/SubtitleEditBeta.zip

**Reply by ard-av** [reply]
works. thank you very much

**Comment by ard-av** [comment]
![Screenshot_2](https://user-images.githubusercontent.com/114281218/193888789-341b4472-4908-4c84-bf44-88c3f7b6ca14.png)
working with file list and clipboard.

**Comment by niksedk** [comment]
Good idea - I've tried to add support for this (but only via Ctrl+v in list view): https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.7/SubtitleEditBeta.zip
Does that work for you?

**Reply by ard-av** [reply]
Ctrl+V works. Thank you. But it's not obvious. Few would guess that this is possible. The effect of the buttons would be greater.

**Reply by ard-av** [reply]
![Screenshot_5](https://user-images.githubusercontent.com/114281218/193907318-2241822f-0c64-4da3-9e2a-7bbb20eed58c.png)
![Screenshot_6](https://user-images.githubusercontent.com/114281218/193907322-15cd633d-bceb-4aa9-a848-6817aedc216c.png)
